src/index.html

```diff
@@ -15,8 +15,109 @@
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossoriginn="anonymus"
```

Comment:
There is a typo in the attribute name `crossoriginn`. It should be `crossorigin` to correctly specify the CORS settings for the preconnect link.